docs/source/cli/config.rst

```diff
@@ -49,6 +49,12 @@ Note: ``config`` is not a command as such, just a help topic (there is no ``git
     If you want the worktree to have its own branch layout file (located under ``.git/worktrees/.../machete``),
     set ``git config machete.worktree.useTopLevelMacheteFile false``.
 
+``machete.squashMergeDetection``:
+    Controls the algorithm used to detect squash merges. Possible values are:
+    * ``none``: No squash merge/rebase detection.
+    * ``simple``: Compares the tree state of the merge commit with the tree state of the upstream branch. This detects squash merges/rebases as long as there was not any commit on the upstream branch since the last common commit.
```

Comment:
```suggestion
    * ``simple``: Compares the tree state of the merge commit with the tree state of the upstream branch. This detects squash merges/rebases as long as there was no commit on the upstream branch since the last common commit.
```